pkg/generator/fluentd/conf.go

```diff
@@ -0,0 +1,106 @@
+package fluentd
+
+import (
+	"encoding/json"
+	"errors"
+	"net/url"
+
+	logging "github.com/openshift/cluster-logging-operator/pkg/apis/logging/v1"
+	. "github.com/openshift/cluster-logging-operator/pkg/generator"
+	corev1 "k8s.io/api/core/v1"
+)
+
+var (
+	ErrNoValidInputs    = errors.New("No valid inputs found in ClusterLogForwarder")
+	ErrNoOutputs        = errors.New("No outputs defined in ClusterLogForwarder")
+	ErrInvalidOutputURL = errors.New("Invalid URL in one of the outputs in ClusterLogForwarder")
```

Comment:
If this is intended to be user-visible, it would be nice to say which output has the invalid URL

Reply:
updated to add the output name in the error